### PR TITLE
Add Jakarta EE Specification Versioning, Change, and Deprecation Process

### PR DESCRIPTION
**Pull request by Pandrex247**
Adds the titular document to the Jakarta EE Specification Committee area.

Also adds IntelliJ to gitignore 😊

### CODE REVIEW

File: content/committees/specification/versioning/index.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+
+## Definitions {#definitions}
+
+We define four “realms” for changes. For clarity, “changes” here refers to additions (new methods to existing 
+interfaces, and new interfaces), and removals (removing methods from existing interfaces, and removing interfaces 
+entirely). The four realms for changes are:
+
+* API changes – these are changes to the APIs available to applications, vendor implementations, and wrapper 
+implementations. 
+  * As an example, the removal of `isParmetersProvided` from Expression Language.
```
Comment (oliviergoulet5) [inline content/committees/specification/versioning/index.md:48]:
```suggestion
  * As an example, the removal of `isParametersProvided` from Expression Language.
```

I found a small typo here

**Review by oliviergoulet5 — CHANGES_REQUESTED** [review]
Apart from the typo, the PR looks good!

File: content/committees/specification/versioning/index.md (hunk 1 above)
Comment (Pandrex247) [reply content/committees/specification/versioning/index.md:48]: That's an intentional typo 👍
I'm giving an example of a misspelt method that was removed.

See the _Removals, Deprecations or Backwards Incompatible Changes_ section of [Expression Language 5.0](https://jakarta.ee/specifications/expression-language/5.0/)

> The deprecated, misspelt method MethodExpression.isParmetersProvided() has been removed.

Comment (oliviergoulet5) [reply content/committees/specification/versioning/index.md:48]: Understood!

**Review by oliviergoulet5 — APPROVED** [review]

**Pull request merged**